packages/manager/apps/key-management-service/src/pages/serviceKey/createKey.page.tsx

```diff
@@ -413,7 +413,12 @@ export default function CreateKey() {
                     inline
                     color={ODS_THEME_COLOR_INTENT.primary}
                     onClick={submitCreateKey}
-                    disabled={!!serviceKeyNameError || isPending || undefined}
+                    disabled={
+                      !!serviceKeyNameError ||
+                      keyOperations.length === 0 ||
+                      isPending ||
+                      undefined
+                    }
```

Comment:
**nitpick:** `undefined` can be removed as it does not have any effect. "true || undefined" will result in `true` and "false || undefined" will result in `false`.
```suggestion
                    disabled={
                      !!serviceKeyNameError ||
                      keyOperations.length === 0 ||
                      isPending
                    }
```

Reply:
removing `undefined` broke the disabled prop behavior of the `<OsdsButton />` component: the button is always disabled.